frontend/src/app/main/ui/ds/notifications/shared/notification_pill.cljs

```diff
@@ -0,0 +1,42 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.ui.ds.notifications.shared.notification-pill
+  (:require-macros
+   [app.main.style :as stl])
+  (:require
+   [app.main.ui.ds.foundations.assets.icon :as i]
+   [rumext.v2 :as mf]))
+
+(defn icons-by-level
+  [level]
+  (case level
+    :info i/info
+    :warning i/msg-neutral
+    :error i/delete-text
+    :success i/status-tick
+    level))
```

Comment:
I think this should be:

```clojure
(defn icons-by-level
  [level]
  (case level
    :info i/info
    :warning i/msg-neutral
    :error i/delete-text
    :success i/status-tick
    i/msg-neutral))
```

The default value should be of the same type, I mean it should be `nil` or `icon` but not a keyword which is receiving by parameters.

Reply:
You are right, fixing it